src/libproc_macro/lib.rs

```diff
@@ -54,7 +54,7 @@ use syntax::ptr::P;
 /// time!
 #[stable(feature = "proc_macro_lib", since = "1.15.0")]
 pub struct TokenStream {
-    inner: Vec<P<ast::Item>>,
+    inner: TokenStream_,
```

Comment:
Is the convention here usually `TokenStreamInner` rather than something ending with a `_`?

(I don't really know what I'm talking about, mostly wondering)